setup.py

```diff
@@ -15,33 +15,47 @@
     except IndexError:
         raise RuntimeError('Unable to determine version.')
 
-install_requires = [
-    'httptools>=0.0.9',
-    'ujson>=1.35',
-    'aiofiles>=0.3.0',
-    'websockets>=3.2',
-]
+setup_kwargs =  {
+        'name': 'sanic',
```

Comment:
I can understand the reasoning but I don't want to add another dependency in our development pipeline. 

I'll look into Hound CI but chances are it'll stay how it is. Also we could look into something like PEP8 Speaks or something else.